packages/kotti-input-number/src/InputNumber.vue

```diff
@@ -97,23 +108,23 @@ export default {
 		isDecrementEnabled() {
 			return (
 				!this.disabled &&
-				(this.currentValueNumber === null ||
-					isInRange({
-						max: null,
-						min: this.min,
-						value: this.currentValueNumber - this.step,
-					}))
+				isInRange({
```

Comment:
no need for an internal currentValueNumber anymore --> since this.value is now the source of truth, and immediate watcher exists 
the null check is not necessary b/c we have a null check to the value property passed to the isInRange func (see above comment on func)